packages/web-components/fast-foundation/src/toolbar/toolbar.template.ts

```diff
@@ -0,0 +1,34 @@
+import { elements, html, slotted } from "@microsoft/fast-element";
+import type { ViewTemplate } from "@microsoft/fast-element";
+import { endTemplate, startTemplate } from "../patterns";
+import type { Toolbar } from "./toolbar";
+
+/**
+ * The template for the {@link @microsoft/fast-foundation#(Toolbar:class)} component.
+ *
+ * @public
+ */
+export const ToolbarTemplate: ViewTemplate<Toolbar> = html`
+    <template
+        aria-label="${x => x.ariaLabel}"
+        aria-labelledby="${x => x.ariaLabelledby}"
```

Comment:
Is there a reason developers can't set the label and labelledby attributes themselves since this is on the outer node of the component?